site/viamguides/src/cv-checkout/cv-checkout.md

```diff
@@ -0,0 +1,257 @@
+ <img src="assets/peach5.jpeg" alt="one more angle san pellegrino peach" width="200" />
+ <img src="assets/blueberry1.jpeg" alt="blueberry topo chico" width="200" />
+ <img src="assets/blueberry5.jpeg" alt="different angle blueberry topo chico" width="200" />
+ <img src="assets/blueberry10.jpeg" alt="one more angle blueberry topo chico" width="200" />
+ <img src="assets/spindrift1.jpeg" alt="spindrift" width="200" />
+ <img src="assets/spindrift2.jpeg" alt="different angle spindrift" width="200" />
+ <img src="assets/spindrift3.jpeg" alt="one more angle spindrift" width="200" />
+
+
+Phew, that was a lot, but your custom model will thank you! Let's make our images smarter by annotating them in the next step.
+
+<!-- ------------------------ -->
+
+## Annotate your dataset
+Duration: 10
+
+Having images to train a model is a good start. However, they won't be useful unless TensorFlow has a bit more information to work with. In this step, you'll draw bounding boxes around your beverages and label them accordingly.
+
+1. In [the Viam app](https://app.viam.com/data/datasets), find the **DATASETS** tab.
+1. Click on the name of your dataset, for example `beverages`:
+   <br>
+   <img src="assets/selectBeverages.png" alt="datasets overview" width="350" />
+1. Here, you'll see all of the images you've captured, neatly grouped into its own space.
+   ![beverages dataset overview](assets/datasetOverview.png)
+1. Select one image from the dataset. A side panel will appear on the right-hand side. You can see details about this image, such as any objects annotated, associated tags, which datasets the image belongs to, among other details. Click on the **Annotate** button in this panel.
+   ![dataset side panel when single image selected](assets/selectSingleImage.png)
+1. The selected image opens to a larger screen. To detect an object within an image, a label must be given. Create an appropriate label for the beverage you have selected, for example `spindrift_pog`:
+    ![image label creation - spindrift_pog](assets/createLabel.png)  
+1. With the appropriate label now chosen, hold the _Command_ or _Windows_ key down while you use your mouse to draw a bounding box around your beverage.
+   ![gif of bounding box being drawn around drink](assets/annotation.gif)
+1. In the **OBJECTS** panel on the right, you'll see your beverage listed, with an object count of `1`. If you hover over this item, you'll see the `spindrift_pog` label appear in the image and the bounding box fill with color.
+   ![drink annotated](assets/drinkAnnotated.png)
+1. Repeat this for the rest of the images that match the label. You can quickly navigate between images by pressing the `>` (right arrow) or `<` (left arrow) keys on your keyboard.
+1. Once you get to a new beverage, create another descriptive label, draw the bounding box, and repeat for the rest of the images of the same beverage. Double check that each image only has one label and detects the correct beverage!
```

Comment:
Consider calling this out as an aside negative / positive. I didn't know you could only have one label per image. I also drew multiple bounding boxes in an image, is that allowed or not preferred?